### PR TITLE
Go Integration

### PR DESCRIPTION
**Pull request by tstrass**
Use `go-cda-tools` gem for import and export.

### CODE REVIEW

**Comment by okeefm** [comment]
Looks like there are some issues the test suite is picking up with `export_with_ffi`

**Comment by tstrass** [comment]
@okeefm that commit should fix it. had to run a `bundle update` for the go-cda-tools gem.
There will still be a rubocop error for that one commented line, but @dczulada is changing that soon.

**Comment by tstrass** [comment]
New Travis failures are now due to out-of-date linux binary in go-cda-tools. That update should be in soon and then we can update the dependency here again.

File: lib/cypress/patient_zipper.rb [hunk 1]
```diff
@@ -22,8 +22,10 @@ def export(patient)
   end
 
   class QRDAExporter
-    C3EXPORTER = HealthDataStandards::Export::Cat1.new('r3')
-    C3_1EXPORTER = HealthDataStandards::Export::Cat1.new('r3_1')
+    C3EXPORTER = GoCDATools::Export::GoExporter.instance
+    C3_1EXPORTER = GoCDATools::Export::GoExporter.instance
```
Comment (tstrass) [inline lib/cypress/patient_zipper.rb]: I think we actually do need them to be 3 different instances of the exporter. not positive but I think we should leave it like this for now. And I think Matt's future performance improvements will make this more efficient.

**Comment by okeefm** [comment]
Do you want to squash these commits down into 1/a few commits?

**Comment by tstrass** [comment]
Not yet. It looks like we want to bump go-cda-tools again so i'll take care of it

**Comment by tstrass** [comment]
🎉

**Pull request merged**